datadog_checks_base/datadog_checks/base/stubs/aggregator.py

```diff
@@ -293,6 +293,57 @@ def assert_all_metrics_covered(self):
             missing_metrics = self.not_asserted()
         assert self.metrics_asserted_pct >= 100.0, 'Missing metrics: {}'.format(missing_metrics)
 
+    def assert_no_duplicate_all(self):
+        """
+        Assert no duplicate metrics and service checks have been submitted.
```

Comment:
What is considered a duplicate metric or service check?

Reply:
Sure, just improve the doc for `assert_no_duplicate_metrics` and `assert_no_duplicate_service_checks`.

I think it's not needed for `assert_no_duplicate_all`, since it just call the other two.